pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
 	core_v1 "k8s.io/api/core/v1"
```

Comment:
Review and improve all comments.

```
	// Parameters holds parameters for the snapshotter.
	// These values are opaque to the system and are passed directly		// These values are opaque to the system and are passed directly
	// to the snapshotter.		// to the snapshotter.
	// +optional		// +optional
	Parameters map[string]string `json:"parameters,omitempty" protobuf:"bytes,3,rep,name=parameters"`		Parameters map[string]string `json:"parameters,omitempty" protobuf:"bytes,3,rep,name=parameters"`
```

e.g. "...storage provider parameters"

Reply:
P0